.travis.yml

```diff
@@ -9,3 +9,4 @@ install:
 
 script:
  - make lint
+ - make test
```

Comment:
Can you just make a note of this in the Readme Development section as well? Something that tells future contributors should run tests locally to avoid regressions (along with lint'ing).

Reply:
Sure 👍